### PR TITLE
[MeshingApplication] Updating and cleaning up the tests

### PR DESCRIPTION
**Pull request by loumalouomega**
Some clean up and update of the tests

### CODE REVIEW

File: applications/MeshingApplication/tests/cpp_tests/test_integration_values_extrapolation_to_nodes_process.cpp [hunk 1]
```diff
@@ -34,204 +34,111 @@ namespace Kratos
...
+//
+//             gid_io.InitializeMesh(label);
+//             gid_io.WriteMesh(rModelPart.GetMesh());
+//             gid_io.FinalizeMesh();
+//             gid_io.InitializeResults(label, rModelPart.GetMesh());
+//             auto this_var = KratosComponents<Variable<double>>::Get("REFERENCE_DEFORMATION_GRADIENT_DETERMINANT");
+//             gid_io.PrintOnGaussPoints(this_var, rModelPart, label);
+//             gid_io.WriteNodalResultsNonHistorical(this_var, rModelPart.Nodes(), label);
+//             gid_io.WriteNodalResultsNonHistorical(NODAL_AREA, rModelPart.Nodes(), label);
+//         }
```
Comment (AlejandroCornejo) [inline applications/MeshingApplication/tests/cpp_tests/test_integration_values_extrapolation_to_nodes_process.cpp:51]: Why is this comented? should we erase it?

File: applications/MeshingApplication/tests/cpp_tests/test_metric_process.cpp [hunk 2]
```diff
@@ -35,199 +35,91 @@ namespace Kratos
...
+//             gid_io.WriteMesh(rModelPart.GetMesh());
+//             gid_io.FinalizeMesh();
+//             gid_io.InitializeResults(label, rModelPart.GetMesh());
+//             gid_io.WriteNodalResults(DISPLACEMENT, rModelPart.Nodes(), label, 0);
+//             gid_io.PrintOnGaussPoints(ERROR_INTEGRATION_POINT, rModelPart, label);
+//             gid_io.PrintOnGaussPoints(CAUCHY_STRESS_VECTOR, rModelPart, label);
+//             gid_io.PrintOnGaussPoints(STRAIN_ENERGY, rModelPart, label);
+//             gid_io.WriteNodalResultsNonHistorical(METRIC_TENSOR_2D, rModelPart.Nodes(), label);
+// //             gid_io.WriteNodalResultsNonHistorical(METRIC_TENSOR_3D, rModelPart.Nodes(), label); // NOTE: 6 components not suported, update
+//         }
```
Comment (AlejandroCornejo) [inline applications/MeshingApplication/tests/cpp_tests/test_metric_process.cpp:70]: IDEM

**Review by AlejandroCornejo — CHANGES_REQUESTED** [review]

**Comment by loumalouomega** [comment]
It is for debug. I leave there for when the tests fail it is easier to find
errors

**Review by AlejandroCornejo — APPROVED** [review]
OK then!

**Comment by loumalouomega** [comment]
Thanks

**Comment by philbucher** [comment]
I guess this could also have been achieved with a flag, but by commenting the compilation time decreases 👍

**Pull request merged**